src/StochasticStyles/spawning.jl

```diff
@@ -171,14 +156,22 @@ struct Exact{T} <: SpawningStrategy
     Exact(threshold::T=0.0) where {T} = new{T}(threshold)
 end
 
-@inline function spawn!(s::Exact, w, offdiags::AbstractVector, add, val, boost=1)
+"""
+    spawn!(s::SpawningStrategy, w, column, val, boost)
+
+Perform stochastic spawns to `w` from `column` with `val` walkers. `val * boost`
+controls the number of spawns performed.
+
+See [`SpawningStrategy`](@ref).
+"""
+@inline function spawn!(s::Exact, w, column, val, boost=1)
     T = valtype(w)
-    spawns = sum(offdiags; init=real(zero(T))) do (new_add, mat_elem)
+    spawns = sum(offdiagonals(column); init=real(zero(T))) do (new_add, mat_elem)
         abs(projected_deposit!(
-            w, new_add, val * mat_elem, add => val, s.threshold
+            w, new_add, val * mat_elem, column.address => val, s.threshold
         ))
     end
-    return (length(offdiags), spawns)
+    return (num_offdiagonals(column), spawns)
```

Comment:
If `num_offdiagonals` no longer needs to be precise, it would be more correct to count the spawns in the `sum`. I'm not sure it's worth the trouble and I think we can leave like this.